support-frontend/assets/helpers/abTests/landingPageAbTests.ts

```diff
@@ -0,0 +1,97 @@
+import seedrandom from 'seedrandom';
+import type {
+	LandingPageTest,
+	LandingPageVariant,
+} from '../globalsAndSwitches/landingPageSettings';
+import type { CountryGroupId } from '../internationalisation/countryGroup';
+import type { Participations } from './models';
+import {
+	getSessionParticipations,
+	LANDING_PAGE_PARTICIPATIONS_KEY,
+	setSessionParticipations,
+} from './sessionStorage';
+
+export type LandingPageSelection = LandingPageVariant & { testName: string };
+
+export const fallBackLandingPageSelection: LandingPageSelection = {
+	testName: 'FALLBACK_LANDING_PAGE',
```

Comment:
Will the fallback be hardcoded or will we pull config from DynamoDB? I don't mind it being hard coded as it's problaby quite important that it doesn't change

Reply:
Yes I was thinking we should keep it hardcoded in the client, in case something goes wrong elsewhere

Reply:
Great idea! We could add a code comment explaining that or some sort of warning to not change the template